test/before.test.ts

```diff
@@ -47,6 +47,11 @@ before(async () => {
     ipfsGatewayUrl,
     registryContractAddress: registryAddress,
     maxGasPriceInGwei: 10000,
+    storage: {
+      thirdwebIpfsServerUrl:
```

Comment:
Why is this stuff in this PR? This PR should be **only** about the date resolver, that's it. Can't merge this with these changes.